modules/Classes.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] I strongly suggest that you don't disable the `"no-use-before-define"` rule. Using functions or variables before they are defined can be confusing and most likely will make your code hard to read. Please consider enabling your linter and fixing thee order of declarations! 🤓 🛠️

Reply:
![Eslint](https://user-images.githubusercontent.com/30289772/207419387-f703857e-23e2-488c-a984-2c36cd35a8d0.PNG)
@anagudelogu 
Thank you for your insightful review.
We were instructed to disable some rules.